Makefile

```diff
@@ -2,7 +2,11 @@
 NUM_ADDONS=10
 NUM_THEMES=$(NUM_ADDONS)
 
-COMPOSE_PROJECT_NAME?=addonsserver
+# Get the name of the Makefile's directory for the docker container base name.
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
```

Comment:
Definitely copy/pasted from http://stackoverflow.com/questions/18136918/how-to-get-current-directory-of-your-makefile.